docs/src/Table/index.js

```diff
@@ -261,44 +261,40 @@ class TableExample extends React.Component {
   // Optional colgroup component.
   colGroup: PropTypes.object,
 
-  // Settings for each column in table.
+  // Define how columns should be rendered and if they are sortable.
   columns: PropTypes.arrayOf(
     PropTypes.shape({
-      // Attributes to pass down to each column item.
-      attributes: React.PropTypes.object,
-
-      // Class to give to each column item. Can be a function to programmatically create a class.
```

Comment:
Maybe we should add these comments back into the table instead of removing them from here, so they are still identical, but still provide useful information